three-pillars/README.md

```diff
@@ -20,21 +20,21 @@ At Dev Academy, we have codes of conduct that support an optimal learning enviro
 
 Part of being in integrity also means that when you fall out of it for whatever reason (such as being a human being) you own up to it.  Most of the time all it takes is an acknowledgment of being out of integrity and everybody can move on.
 
-## The second agreement: Kindness
+## Kindness
 We also expect every person to be kind to one another. Kindness does not mean "nice;" they are actually not the same thing. Being kind does mean being friendly, generous, and considerate, but it does not preclude giving constructive, truthful feedback as long as it is <strong>A</strong>ctionable, <strong>S</strong>pecific, and <strong>K</strong>ind (ASK).
```

Comment:
I don't think we need to bold the first letter of each word, but even if we do, we can use markdown.